mmv1/third_party/terraform/tests/resource_redis_instance_test.go

```diff
@@ -40,6 +40,148 @@ func TestAccRedisInstance_update(t *testing.T) {
 	})
 }
 
+func TestAccRedisInstance_updateReadReplicasMode(t *testing.T) {
```

Comment:
Could you add a comment in this test case somewhere explaining what is being tested? If I understand correctly, the purpose of this test is to ensure that changing read replicas mode from unspecified -> enabled works and doesn't destroy / recreate the instance?

Reply:
Added the comment